core/platforms/packaging/bareos.spec

```diff
@@ -422,6 +429,17 @@ Summary:    Python plugin for Bareos Director daemon
 Group:      Productivity/Archiving/Backup
 Requires:   bareos-director = %{version}
 
+%if 0%{?enable_grpc}
+%package    filedaemon-grpc-python3-plugin
+Summary:    Python plugin for Bareos File daemon
+Group:      Productivity/Archiving/Backup
+Requires:   bareos-filedaemon = %{version}
+Requires:   bareos-filedaemon-python-plugins-common = %{version}
+Provides:   bareos-filedaemon-grpc-python-plugin
+Obsoletes:  bareos-filedaemon-grpc-python-plugin <= %{version}
+Provides:   bareos-filedaemon-python-plugin
+Obsoletes:  bareos-filedaemon-python-plugin <= %{version}
```

Comment:
This is not provided by the grpc plugin.  The grpc plugin requires them instead.